src/components/Redirection.js

```diff
@@ -1,23 +1,25 @@
-import Qs from 'qs';
-import { useEffect } from 'react';
-import { useLocation } from 'react-router';
+import PropTypes from 'prop-types';
+
+import { RedirectionContent } from '@graasp/ui';
+import { getUrlForRedirection, redirectToSavedUrl } from '@graasp/utils';
 
 import { GRAASP_COMPOSE_HOST } from '../config/constants';
 import { hooks } from '../config/queryClient';
 
-function Redirection({ children }) {
+const Redirection = ({ children }) => {
   const { data: member } = hooks.useCurrentMember();
-  const location = useLocation();
-
-  useEffect(() => {
-    if (member?.get('id')) {
-      const queryString = Qs.parse(location.search, {
```

Comment:
Was this code moved to utils? If yes, doesn't this allow injection of arbitrary redirections (i.e. passing any valid URL to the `search` GET parameter would allow any kind of redirection)?

Reply:
We don't use search from querystring anymore 👍